src/server.c

```diff
@@ -732,14 +732,14 @@ int clientsCronResizeOutputBuffer(client *c) {
         server.stat_reply_buffer_expends++;
     }
 
-    /* reset the peak value each 5 seconds. in case the client will be idle
+    /* reset the peak value each server.reply_buffer_peak_reset_time seconds. in case the client will be idle
      * it will start to shrink.
      */
     if (server.reply_buffer_peak_reset_time >=0 &&
-        server.unixtime - c->buf_peak_last_reset_time >= server.reply_buffer_peak_reset_time)
+            now_ms - c->buf_peak_last_reset_time >= server.reply_buffer_peak_reset_time)
     {
         c->buf_peak = c->bufpos;
-        c->buf_peak_last_reset_time = server.unixtime;
+        c->buf_peak_last_reset_time = now_ms;
```

Comment:
i think i meant to use `server.mstime` instead of `server.unixtime`
i'm not sure why clientsCron samples `mstime()` on it's own.. but well, don't care either way.

Reply:
I initially did use the server mstime. but felt no_ms would be even more precise :)